src/scripts/tests/simpleKingMoves.spec.ts

```
@@ -0,0 +1,32 @@
+    const board = new Board();
+    const piece = new Piece(false, new Vector2(0, 0), true);
+    board.addPiece(piece);
+    board.addPiece(new Piece(false, new Vector2(1, 2)));
+    board.addPiece(new Piece(false, new Vector2(0, 2)));
+    board.addPiece(new Piece(false, new Vector2(2, 2)));
+    board.addPiece(new Piece(false, new Vector2(2, 1)));
+    board.addPiece(new Piece(false, new Vector2(2, 0)));
+
+    const moves = MoveCalculator.calculateMoves(board, piece);
+
+    assert.lengthOf(moves!, 3);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)